qemu/tests/block_copy.py

```diff
@@ -426,6 +419,7 @@ def create_file(self, file_name):
         params = self.params
         session = self.get_session()
         file_create_cmd = params.get("create_command", "touch FILE")
+        file_create_cmd = utils_misc.set_winutils_letter(session, file_create_cmd)
```

Comment:
When it comes to windows guest, params["create_command"] is not optional any more.
Otherwise `utils_misc.set_winutils_letter` deos not do anything.
I think `file_create_cmd = utils_misc.set_winutils_letter(session, params["create_command"])` may be better.

Reply:
@hereischen Thanks, that's true, however I just want to keep a default option for Linux, for user do not need to specify it for Linux. set_winutils_letter() will do nothing for Linux anyway.